perception/sub8_perception/nodes/path_marker_finder.py

```diff
@@ -37,6 +38,11 @@ class PathMarkerFinder():
     # Model of four corners of path marker, centered around 0 in meters
     LENGTH = 0.6096 # Longer side of path marker in meters
     WIDTH = 0.0762 # Shorter Side
+
+    # Demensions of the prop we made for testing
```

Comment:
Still, don't think it would be useful to have the length and width as part of the service request? It would allow us to use arbitrarily sized strips, especially useful for testing.